src/vehicle.h

```diff
@@ -1331,6 +1331,10 @@ class vehicle
         turret_data turret_query( const tripoint &pos );
         turret_data turret_query( const tripoint &pos ) const;
 
+        /** Returns true if any part on the tile the turret is installed on has the MANUAL flag. */
+        bool is_manual_turret( vehicle_part &pt );
```

Comment:
```suggestion
        bool is_manual_turret( vehicle_part &pt ) const;
```

shouldn't this be marked as `const` too?

Reply:
note that relevant code changes should be made too.

Reply:
Oh shit, hecc.